wiki/fr/fr-Onboarding.md

```diff
@@ -29,29 +35,34 @@ Maintenant vous allez voir ceci :
+Maintenant que vous avez configuré matériel et logiciel, vous pouvez utiliser le bouton « Se connecter » dans la fenêtre principale de Jamulus pour rejoindre un serveur. Une fenêtre s'ouvre alors :
+
+![Connection Setup](https://user-images.githubusercontent.com/20726856/102825226-a1b92c00-43de-11eb-9aa9-21d7a8576f5c.png)
+
+Cette fenetre montre les *serveurs regroupés par genre*. Pour choisir un autre genre, sélectionnez-en un dans la liste en haut à gauche. Après avoir sélectionné un genre et un serveur, rejoignez le serveur en cliquant sur « Se connecter ». Si vous rencontrez des problèmes ici, ils sont très probablement dus à un équipement audio mal configuré ou à des problèmes de réseau. Consultez la [page de dépannage du client](Client-Troubleshooting).
 
 ### Le premier bœuf
 Félicitations ! Vous pouvez maintenant commencer à bœuffer avec Jamulus. Les chariots que vous voyez sur la droite sont votre propre mixage personnel. Tout ce que vous modifiez ici agit sur ce que vous entendez, mais n'affectera pas les autres utilisateurs. Si vous déplacez un chariot vers le bas, cet utilisateur sera moins fort, si vous le déplacez vers le haut, cet utilisateur sera plus fort pour vous.
 
 ![Mixeur principal de Jamulus connecté](https://user-images.githubusercontent.com/9108457/101895820-55811700-3ba8-11eb-9945-7923c8a92c60.png)
 
-Si vous ne voulez pas que les autres vous entendent votre audio, cliquez sur « Me silencer » (à gauche de votre mixage personnel) ce qui empêchera l'envoi de votre audio aux autres musiciens.
+Si vous ne voulez pas que les autres entendent votre audio, cliquez sur « Me silencer » (à gauche de votre mixage personnel) ce qui empêchera l'envoi de votre audio aux autres musiciens.
 
 Pendant que nous y sommes : « Paramètres » ouvre, évidement, la fenêtre des paramètres avec tous les paramètres audio que vous voudrez probablement modifier pour vos besoins.
 
 De plus amples informations sur l'utilisation de Jamulus peuvent être trouvées dans le [manuel du logiciel](Software-Manual).
 
 ## Avancé
-<!-- Bien sûr, vous pouvez faire beaucoup plus avec Jamulus, comme configurer un serveur privé, faire des concerts en ligne, … Jetez un coup d'œil sur ce site ! La communauté publie des guides, trucs, astuces dans la [base de connaissance](/kb/). --> 
+<!-- Bien sûr, vous pouvez faire beaucoup plus avec Jamulus, comme configurer un serveur privé, faire des concerts en ligne, … Jetez un coup d'œil sur ce site ! La communauté publie des guides, trucs et astuces dans la [base de connaissance](/kb/). --> 
```

Comment:
Are you sure this is still commented?

Reply:
Yes it's already translated but changing only "," by "et" sounds better for me.